src/index.rs

```diff
@@ -55,17 +64,15 @@ impl Index {
             unsafe { raw::RediSearch_CreateField(self.inner, name.as_ptr(), ftype.bits, fopt) };
         unsafe {
             raw::RediSearch_TextFieldSetWeight(self.inner, field_id, weight);
-            if let Some(options) = tag_options {
-                if let Some(sperator) = options.tag_separator {
-                    raw::RediSearch_TagFieldSetSeparator(self.inner, field_id, sperator as i8);
-                }
-                if options.tag_case_sensitive {
-                    raw::RediSearch_TagFieldSetCaseSensitive(
-                        self.inner,
-                        field_id,
-                        options.tag_case_sensitive as i32,
-                    );
-                }
+            if let Some(sperator) = tag_options.tag_separator {
```

Comment:
The typo is not fixed yet... `sperator` => `separator`

Reply:
:+1: